Content.Shared/_RMC14/Xenonids/Parasite/ParasiteResistanceComponent.cs

```diff
@@ -0,0 +1,11 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared._RMC14.Xenonids.Parasite;
+
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
+[Access(typeof(SharedXenoParasiteSystem))]
+public sealed partial class ParasiteResistanceComponent : Component
+{
+    [DataField, AutoNetworkedField]
+    public float Probability = 0.5f;
```

Comment:
From 13 this is a counter not a probability
The B18 and grenadier helmet resists 6 hugs
Scout helmet resists 0 (lol)
Rest are all 0